scalameta/semanticdb-scalac/src/main/scala/scala/meta/internal/semanticdb/DocumentOps.scala

```diff
@@ -250,12 +257,16 @@ trait DocumentOps { self: DatabaseOps =>
               // Unfortunately, this is often not the case as demonstrated by a bunch of cases above and below.
               if (tryMpos(gstart, gend)) return
 
+              pprint.log(gtree)
               gtree match {
                 case gtree: g.ValDef if gtree.symbol == gtree.symbol.owner.thisSym =>
                   tryMstart(gstart)
                 case gtree: g.MemberDef if gtree.symbol.isSynthetic || gtree.symbol.isArtifact =>
                 // NOTE: never interested in synthetics except for the ones above
                 case gtree: g.PackageDef =>
+                  if (config.members.isAll) {
+                    members(gtree.symbol.toSemantic) = gtree.pid.tpe.toMembers
+                  }
```

Comment:
How about we centralize this logic in `saveDenotation`?

Reply:
We don't want to persist members for all denotations, only packages/wildcard imports/Template.inits. I centralized the `toMembers` logic, but I don't think it's worth abstracting over this here since there are minor nuances in each case (import/package/...)